scripts/vagrant/zeppelin-dev/roles/nodejs/meta/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+galaxy_info:
+  author: Mark Wolfe
+  description: Installs the NodeSource Node.js binary packages
+  company: NodeSource
+  license: MIT
```

Comment:
is this not from you and is separately licensed? you would need to include link to source and license under /licenses